dataservice/api/genomic_file/models.py

```diff
@@ -1,12 +1,31 @@
+import requests
+
+from flask import abort, current_app
+from requests.exceptions import HTTPError
+from sqlalchemy import event
+from sqlalchemy.dialects.postgresql import UUID
+
 from dataservice.extensions import db
 from dataservice.api.common.model import Base, KfId
-from sqlalchemy.dialects.postgresql import UUID
 
 
 class GenomicFile(db.Model, Base):
```

Comment:
Could we move all the code that interfaces with the indexd service out of the ORM class? Maybe we could create a new package `dataservice/external` and create an `indexd` or `gen3` module or something. All of the code to talk  to indexd could go in that module. Then you would call that module in the GenomicFileAPI class. Keeps our persistence layer/code clean and completely separate from the http layer.

Reply:
Yeah, I wanted to do that, but wasn't sure where to put it. It turns out that their [Client](https://github.com/uc-cdis/indexclient) is open which is really what I ought to be using.